docs/content/preview/releases/yba-releases/v2024.1.md

```diff
@@ -27,6 +27,73 @@ If you have a Replicated installation, you must migrate from Replicated to YBA I
 YugabyteDB 2024.1.0.0 and newer releases do not support v7 Linux versions (CentOS7, Red Hat Enterprise Linux 7, Oracle Enterprise Linux 7.x), Amazon Linux 2, and Ubuntu 18. If you're currently using one of these Linux versions, upgrade to a supported OS version before installing YugabyteDB v2024.1.0. Refer to [Operating system support](/stable/reference/configuration/operating-systems/) for the complete list of supported operating systems.
 {{</warning>}}
 
+## v2024.1.3.0 - October 8, 2024 {#v2024.1.3.0}
+
+**Build:** `2024.1.3.0-b105`
+
+**Third-party licenses:** [YugabyteDB](https://downloads.yugabyte.com/releases/2024.1.3.0/yugabytedb-2024.1.3.0-b105-third-party-licenses.html), [YugabyteDB Anywhere](https://downloads.yugabyte.com/releases/2024.1.3.0/yugabytedb-anywhere-2024.1.3.0-b105-third-party-licenses.html)
+
+### Download
+
+<ul class="nav yb-pills">
+ <li>
+   <a href="https://downloads.yugabyte.com/releases/2024.1.3.0/yba_installer_full-2024.1.3.0-b105-linux-x86_64.tar.gz">
+     <i class="fa-brands fa-linux"></i>
+     <span>Linux x86</span>
+   </a>
+ </li>
+</ul>
+
+### New features
+
+* [Validation of Certificate Authority settings](/stable/yugabyte-platform/security/authentication/oidc-authentication-aad/). When configuring an external Certificate Authority to manage SSL certs used in Encryption in Transit (EIT), YugabyteDB Anywhere now validates the settings up front, thereby improving usability.
```

Comment:
```suggestion
* [Validation of Certificate Authority settings](/stable/yugabyte-platform/security/authentication/oidc-authentication-aad/). When configuring an external Certificate Authority to manage SSL certificates used for encryption in transit, YugabyteDB Anywhere now validates the settings up front, improving usability.
```

Reply:
Link seems unrelated. Maybe stable/yugabyte-platform/create-deployments/create-universe-multi-zone/#encryption-settings?

Reply:
You’re right.. its an incorrect IDEA. Changing it. Thanks.